content/en/docs/ops/configuration/telemetry/monitoring-multicluster-prometheus/index.md

```diff
@@ -0,0 +1,164 @@
+
+Our recommendation for multicluster monitoring of Istio with Prometheus is built upon the foundation of Prometheus
+[hierarchical federation](https://prometheus.io/docs/prometheus/latest/federation/#hierarchical-federation).
+Cluster-local Istio-deployed Prometheus instances will act as initial collectors which then federate up to a production
+mesh-wide Prometheus instance. That mesh-wide Prometheus can either live outside of all of the clusters from the mesh
+(aka external), or in one of the clusters within the mesh.
+
+## Multicluster Istio Setup
+
+There are a couple of [multicluster deployment models](/docs/ops/deployment/deployment-models/#multiple-clusters)
+supported by Istio. You can follow the [multicluster installation](/docs/setup/install/multicluster/) section to setup
+your multicluster Istio. For the purposes of this guide, any of those approaches will work, with the following
+caveat:
+
+**Ensure that a cluster-local Istio Prometheus instance is installed in each cluster.**
+
+Individual Istio deployments of Prometheus in each cluster will be used to form the basis of cross-cluster monitoring by
+way of federation to either an external production-ready Prometheus or a production-ready instance of Prometheus
+running on one of the clusters.
+
+For multicluster deployments that use the `remote` profile, this will require adding the following to the istioctl
+manifest command:
+
+{{< text bash >}}
+$ istioctl manifest apply -f --set addonComponents.prometheus.enabled=true
+{{< /text >}}
+
+Validate that you have an instance of Prometheus running in each cluster:
+
+{{< text bash >}}
+$ kubectl -n istio-system get services prometheus
+NAME         TYPE        CLUSTER-IP   EXTERNAL-IP   PORT(S)    AGE
+prometheus   ClusterIP   10.8.4.109   <none>        9090/TCP   20h
+{{< /text >}}
+
+## Configure Prometheus Federation
+
+### External Production Prometheus
+
+There are several reasons why you may want to have a Prometheus instance running outside of your Istio deployment.
+Perhaps you want long-term monitoring disjoint from the cluster being monitored. Perhaps you want to monitor multiple
+separate meshes in a single place. Or maybe you have other motivations. Whatever your reason is, you’ll need some special
+configurations to make it all work.
+
+{{< image width="80%"
+    link="./external-production-prometheus.svg"
+    alt="Architecture of external Production Prometheus for monitoring multicluster Istio."
+    caption="External Production Prometheus for monitoring multicluster Istio"
+    >}}
+
+*Note:* This guide does not cover securing access to Prometheus. It demonstrates providing connectivity to cluster-local Prometheus
+instances in a few simple ways. For production use cases, it is recommended to secure access to each Prometheus endpoint
+with HTTPS, as well as taking appropriate precautions such as using an internal load-balancer instead of a publicly-accessible
+endpoint and/or properly configuring firewall rules.
```

Comment:
```suggestion
{{< warning >}}
This guide does not cover securing access to Prometheus. It demonstrates providing connectivity to cluster-local Prometheus
instances in a few simple ways. For production use cases, it is recommended to secure access to each Prometheus endpoint
with HTTPS, as well as taking appropriate precautions such as using an internal load-balancer instead of a publicly-accessible
endpoint and/or properly configuring firewall rules.
{{< /warning >}}
```

Reply:
done